src/uct/api/uct.h

```diff
@@ -812,10 +813,10 @@ UCT_INLINE_API ucs_status_t uct_ep_put_short(uct_ep_h ep, const void *buffer, un
  * @brief
  */
 UCT_INLINE_API ucs_status_t uct_ep_put_bcopy(uct_ep_h ep, uct_pack_callback_t pack_cb,
-                                             void *arg, size_t length, uint64_t remote_addr,
```

Comment:
not always, for example when packing custom datatype the length is not needed (ucp always passes max_bcopy to the packer)